cfgov/cfgov/urls.py

```diff
@@ -412,7 +429,8 @@
         url(r'^(?i)obtener-respuestas/api/autocomplete-(?P<language>es)/$',
             ask_autocomplete, name='ask-autocomplete-es'),
     ]
-    urlpatterns += ask_patterns
+
+urlpatterns += ask_patterns
```

Comment:
@higs4281 That's not quite right, and you've spotted a flaw here. For the shared URLs between knowledgebase and ask_cfpb we'll have to have a `flagged_url` with a fallback.